controller.js

```diff
@@ -0,0 +1,16 @@
+class PokemonController {
+    constructor(pokemonService) {
+        this.pokemonService = pokemonService;
+    }
+
+    async getPokemons(req, res) {
```

Comment:
as this function is basically returning the complete list pokemons, name it `getAllPokemons`. Also I am telling you this because we will add new routes for quering.
```suggestion
    async getAllPokemons(req, res) {
```